app/src/main/java/com/wearerommies/roomie/presentation/ui/home/HomeViewModel.kt

```diff
@@ -48,6 +48,12 @@ class HomeViewModel @Inject constructor(
         }
     }
 
+    fun navigateToMap() {
+        viewModelScope.launch {
+            _sideEffect.emit(HomeSideEffect.NavigateToMap)
+        }
+    }
```

Comment:
요거 봤는지 모르겠지만...
```suggestion
fun navigateToMap()  = viewModelScope.launch {
            _sideEffect.emit(HomeSideEffect.NavigateToMap)
}
``` 

이렇게도 가능해요!